packages/pages/src/upgrade/upgrade.ts

```diff
@@ -1,19 +1,14 @@
 import { Command } from "commander";
-import { updatePages } from "./pages_updater.js";
-import { migrateConfigs } from "./migrate_config.js";
-import { ProjectStructure } from "../common/src/project/structure.js";
+import { updatePages } from "./pagesUpdater.js";
+import { migrateConfigs } from "./migrateConfig.js";
 import { templatesHandler } from "../generate/templates/templates.js";
 import { artifactsHandler } from "../generate/artifacts/artifacts.js";
-import { cleanConfigs } from "./clean_configs.js";
 
 const handler = async ({ scope }: { scope: string }) => {
-  const scoped = { scope: scope };
-  const projectStructure = await ProjectStructure.init(scoped);
-  await updatePages(projectStructure);
-  await migrateConfigs(projectStructure);
-  await templatesHandler(scoped);
-  await artifactsHandler(scoped);
-  await cleanConfigs(projectStructure);
+  await updatePages(scope);
+  await migrateConfigs(scope);
+  await templatesHandler({ scope: scope });
```

Comment:
It's weird that the functions take in different things, an object that contains a scope or just scope directly. Also, it's preferable to just init ProjectStructure once like you had before instead of in each handler.

Reply:
I think the weird scope arguments are an artifact of how Commander handles option arguments.

I had changed it because we have to init projectStructure multiple times during the recursive migrate, but I changed  upgrade.ts it to use less inits().